doc/api/events.md

```diff
@@ -142,7 +142,7 @@ myEmitter.emit('error', new Error('whoops!'));
 
 To guard against crashing the Node.js process, a listener can be registered
 on the [`process` object's `uncaughtException` event][] or the [`domain`][] module
-can be used. (_Note, however, that the `domain` module has been deprecated_)
+can be used. _(Note, however, that the `domain` module has been deprecated.)_
```

Comment:
And what about this case, to my perception, this is not a note as in "Note:", but rather just a regular sentence. I fixed the stylistic mistakes while I was here, though.